### PR TITLE
<PreviewWidget/>- Blank type

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary

Related to https://github.com/wix/wix-style-react/issues/3833

New component: https://zpl.io/2jMDQLr

This PR is related to the `blank` type without the custom background color. The custom background color will be implemented in different PR. 

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [x] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [x] Story
  - [x] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

File: src/Preview/Preview.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    const {
+      dataHook,
+      type,
+      backgroundColor,
+      borderType,
+      children,
+    } = this.props;
+
+    return (
+      <div {...style('root', { backgroundColor })} data-hook={dataHook}>
```
Comment (mosheKerbel) [inline src/Preview/Preview.js]:
```suggestion
      <div {...style('root', { backgroundColor }, this.props)} data-hook={dataHook}>
```
passing the 3rd arg lets consumers override the style.

File: src/Preview/Preview.st.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  color2: #eee8e5;
+}
+
+.root{
+  -st-states: backgroundColor(enum(grey, gradient));
+  padding: 30px;
+  display: inline-block;
+}
+
+/* backgroundColor = grey*/
```
Comment (mosheKerbel) [inline src/Preview/Preview.st.css]: this is obvious, comments is redundant.

File: src/Preview/Preview.st.css [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+.root:backgroundColor(grey){
+  background-color: value(D60);
+}
+
+/* backgroundColor = gradient */
```
Comment (mosheKerbel) [inline src/Preview/Preview.st.css]: this is obvious, comments is redundant.

File: src/Preview/Preview.st.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+.root:backgroundColor(gradient){
+  background-image: linear-gradient(value(deg), value(color1) , value(color2));
+}
+
+.contentArea {
+  -st-states: borderType(enum(shadow, solid));
+  border-radius: 4px;
+}
+
+/* borderType = shadow  */
```
Comment (mosheKerbel) [inline src/Preview/Preview.st.css]: this is obvious, comments is redundant.

File: src/Preview/Preview.st.css [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+.contentArea:borderType(shadow){
+  box-shadow: value(shadow30);
+}
+
+/* borderType = solid  */
```
Comment (mosheKerbel) [inline src/Preview/Preview.st.css]: this is obvious, comments is redundant.

File: src/Preview/Preview.st.css [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+:import {
+  -st-from: "../Foundation/stylable/shadows.st.css";
+  -st-named: shadow30;
+}
+
+:vars{
+  deg: 307deg;
+  color1: #d4e5f2 10%;
```
Comment (mosheKerbel) [inline src/Preview/Preview.st.css]: these var names are not informative
not sure wether values should even be inside a var

File: testkit/component-definitions.js [hunk 7]
```diff
@@ -275,4 +275,10 @@ export default {
       media: <div />,
     },
   },
+
+  Preview: {
```
Comment (TalZa1810) [inline testkit/component-definitions.js]: I've added this so that the testkit tests won't fail
@mosheKerbel

**Pull request merged**